packages/calling/calling/src/CallingClient/constants.ts

```diff
@@ -66,6 +66,7 @@ export const WEBEX = 'webex';
 export const WEBEX_WEB_CLIENT = 'webex-web-client';
 export const CALLER_ID_FILE = 'CallerId';
 export const UTILS_FILE = 'utils';
+export const CALLING_FILE = 'Calling';
```

Comment:
Should we be doing this.. importing a constant across packages..

Reply:
I am not sure about it. Maybe @arun3528  can comment on it

Reply:
As discussed, removed it